ajax/libs/deferer/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "deferer",
+  "version": "1.0.0",
+  "description": "Redirect traffic for matching referering hosts",
+  "author": "Dominic Dagradi <ddagradi@gmail.com>",
+  "repository": "https://github.com/dominic/deferer",
+  "main": "deferer.js",
+  "license": "MIT",
+  "dependencies": {},
+  "npmName": "deferer",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "*/"
```

Comment:
@dominic, no need to sorry.
And it's okay to open this until maybe you got a number > 50 of watch + star + fork

cc @IonicaBizau